solution_template/createUiDefinition.json

```diff
@@ -296,6 +296,94 @@
               ]
             },
             "visible": "[not(equals(steps('secondStep').sptype,'off'))]"
+          },
+          {
+            "name": "enableArtifactManager",
+            "type": "Microsoft.Common.DropDown",
+            "label": "Enable Azure Native Artiface Manager",
+            "defaultValue": "Yes",
+            "toolTip": "Natively store Jenkins artifacts in Azure storage.",
+            "constraints": {
+              "allowedValues": [
+                {
+                  "label": "Yes",
+                  "value": "yes"
+                },
+                {
+                  "label": "No",
+                  "value": "no"
+                }
+              ]
+            },
+            "visible": true
+          },
+          {
+            "name": "storageAccount",
+            "type": "Microsoft.Common.Section",
+            "label": "Azure Artifact Manager",
+            "elements": [
+              {
+                "name": "storageAccountName",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Storage Account Name",
+                "defaultValue": "",
```

Comment:
with default value? experience like public ip.

Reply:
Not necessary for this value since the storage account does not have specific pattern like IP. Service Principal related variables do not have default values too.